src/app/api/tools/cowswap/orderFlow.ts

```diff
@@ -93,7 +96,28 @@ export async function orderRequestFlow({
   const orderUid = await orderbook.sendOrder(order);
   console.log("Order Posted", orderbook.getOrderLink(orderUid));
 
+  const swapData: SwapFTData = {
+    network: {
+      name: chainId.toString(),
+      icon: "",
+    },
+    type: "swap",
+    tokenIn: {
+      name: quoteRequest.sellToken,
+      icon: "",
+      amount: quoteResponse.quote.sellAmount,
```

Comment:
This is going to be WEI amount and you haven't supplied the token decimals for formatting. If this is a UI component you could formate the values right here. The decimals should be available in buyTokenData and sellTokenData.